kernel/virtio/boot.S

```diff
@@ -27,29 +27,28 @@
  */
 
 #include "multiboot.h"
+#include "../cpu_x86_64.h"
```

Comment:
We can do that if (unlikely) virtio ever gets ported to a non x86_64 architecture.